vulnerabilities/templates/rest_framework/api.html

```diff
@@ -0,0 +1,32 @@
+{% extends "rest_framework/base.html" %}
+
+{% block style %}
+    {{ block.super }}
+    <style>
+        body {background: none;}
+        .navbar {background: #363636; border-top: 3px solid #3298dc;}
+        .navbar-inverse .navbar-brand {color: white;}
+        .nexb-blue
```

Comment:
This line seems problematic.

Reply:
Yeah, it does, good eye, looks like somebody started to define the CSS for `.next-blue` and got distracted, never finished.  @pombredanne , where did this code come from?

Reply:
@tdruez @johnmhoran .nexb-blue is now the name... it is defined in https://github.com/nexB/vulnerablecode/pull/1466/files#diff-9cd3f89490bba2f7d7e1e8174417074cc8f0fbdeaf3b603c0f19f57e0b2e8f2fR15 
is this OK?

Reply:
```
.navbar-inverse .navbar-brand {color: white;}
.nexb-blue
.booleanwidget {width: 90%;}
```

You need to remove the `.nexb-blue` line, it's not a correct syntax.